py/server/deephaven/stream/__init__.py

```diff
@@ -9,14 +9,14 @@
 from deephaven import DHError
 from deephaven.table import Table
 
-_JStreamTableTools = jpy.get_type("io.deephaven.engine.table.impl.StreamTableTools")
+_JBlinkTableTools = jpy.get_type("io.deephaven.engine.table.impl.BlinkTableTools")
 
 
 def stream_to_append_only(table: Table) -> Table:
```

Comment:
Do we want to rename this method? Create a new method, and deprecate this one?

Reply:
Definitely rename. I'm a little iffy on creating a new method and deprecating, but I think doing so would be in-line with my current deprecation strategy.